src/libsyntax/parse/parser.rs

```diff
@@ -4273,7 +4274,9 @@ impl<'a> Parser<'a> {
                 where_clause: WhereClause {
                     id: ast::DUMMY_NODE_ID,
                     predicates: Vec::new(),
-                }
+                },
+                // We lessen by BytePos(1) because self.span is already the element following ">".
```

Comment:
No, For example:

```Rust
fn f<T>(T) {}
```

In here, after the function call, the span is ")" instead of ">". So `self.span` is the following element.